compressor/management/commands/compress.py

```diff
@@ -211,44 +211,54 @@ def compress(self, log=None, **options):
                       "\n\t".join((t.template_name
                                    for t in compressor_nodes.keys())) + "\n")
 
+        contexts = settings.COMPRESS_OFFLINE_CONTEXT
+        if isinstance(contexts, six.string_types):
+            module, function = settings.COMPRESS_OFFLINE_CONTEXT.rsplit('.', 1)
+            contexts = getattr(import_module(module), function)()
```

Comment:
Looking at how it is done in `compressor.cache`, the `AttributeError` is caught anyway. So updating to the same approach should handle all of the error cases.